### PR TITLE
Hi h 24

### PR DESCRIPTION
**Pull request by InnanaDM**

### CODE REVIEW

**Review by imanischult — APPROVED** [review]
Looks good

File: models/user.js [hunk 1]
```diff
@@ -12,10 +12,16 @@ module.exports = function (sequelize, DataTypes) {
             type: DataTypes.STRING,
             allowNull: false
         },
+        displayName: {
```
Comment (Corbineau) [inline models/user.js]: remove the display name here; we won't need that.

File: public/userFriends.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (Corbineau) [inline public/userFriends.html]: question-- what's up with this one? Is this going to be a new layout?

**Review by Corbineau — APPROVED** [review]
All good.

**Pull request merged**